doc/mapping/enum.md

```diff
@@ -8,12 +8,23 @@ However, the Npgsql provider also allows you to map your CLR enums to [database
 
 First, you must specify the PostgreSQL enum type on your model, just like you would with tables, sequences or other databases objects:
 
+# [Version 2.2](#tab/tabid-1)
```

Comment:
This doesn't belongs in master, right? The idea is to have only 2.1 documentation in master, and all 2.2 stuff in dev?